src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -93,4 +97,22 @@ class ElasticSearchDAO(servers:Seq[Authority], indexName: String) extends Search
       case e: Exception => logger.warn(s"ES index '%s' could not be recreated and may be in an unstable state.".format(indexName))
     }
   }
+
+  override def findDocuments(criteria: LibrarySearchParams) : LibrarySearchResponse = {
+    val searchStr =
+      criteria.searchTerm match {
+        case None => ESQuery(Left(new ESMatchAll)).toJson.prettyPrint
+        case Some(searchTerm:String) => ESQuery(Right(ESWildcardObj(searchTerm))).toJson.prettyPrint
+      }
```

Comment:
as a tiny optimization, use `.compactPrint` instead of `.prettyPrint`, so we're sending slightly fewer bytes over the wire - this is only between services, right?